### DISCUSSION TITLE
Thread Router operation in Multiprotocol Environment – Minimum Receive Duty Cycle

### DISCUSSION THREAD

**Discussion by firehills** [openthread/openthread#8850] [Q&A] [answered]
A number of manufacturers (in my case Nordic) are providing multiprotocol implementations where the Rf front end can be multiplexed with other protocols (BLE / Zigbee ….etc) 

In these cases any other protocol activity (eg a BLE scan window) will prevent receipt of internode messages. Since Thread messaging is UDP and inherently lossy, _some_ message loss is expected and in the real world likely to happen. Both the protocol and applications build on top will need to manage this. 

Extreme example:

 I have a BLE/Thread application. The node is configured as a Full End Device.  if I spend 75% of my time scanning BLE, (75ms scan window, 100ms interval), I see the Thread network frequently reconfigure or split. 

Reducing this to 10% scan window, obviously improves the situation, but negatively impacts my BLE app performance. 

**I’m trying to identify if there is either a stated, recommended or theoretical minimum Rx window and interval time a Full Thread Device must be given to maintain the network or alternatively a figure for percentage message loss that still keeps Thread functioning within spec. ??**

Thanks in advance for any advice offered.

**Comment by jwhui** [comment, marked as answer]
I'm not aware of any stated duty-cycle number. Thread assumes routers always have their receiver enabled (when not transmitting). Of course, because Thread performs link-layer retransmissions, it is possible to disable the receiver for short periods of time and recover from any missed packets. That said, decreasing the receiver duty-cycle will further impact performance.

Silicon Labs published a relevant [study on multi-network operation](https://www.silabs.com/documents/public/application-notes/an724-multi-network.pdf). It's not specific to Thread, but the concepts are applicable. In short, decreasing the duty-cycle by a little is OK, but at some point performance will fall off a cliff.
